docs/references/functions/create-variable.md

```diff
@@ -0,0 +1 @@
+Create a new function variable. These variables can be accessed within functions using the `env` object recieved through the payload of an execution.
```

Comment:
```suggestion
Create a new function variable. These variables can be accessed within functions using the `env` object received through the payload of an execution.
```

Reply:
Do we also have a section we can link to in functions doc?

Reply:
@gewenyu99 This is the closest thing we have: https://appwrite.io/docs/functions#environmentVariables
I think we do miss a line or two about adding your own variables.